applications/sasquatch/values-summit.yaml

```diff
@@ -382,6 +390,15 @@ telegraf:
         [ "lsst.obsenv" ]
       debug: true
       offset: newest
+    scimma:
+      enabled: true
+      database: "lsst.scimma"
+      timestamp_format: "unix_ms"
+      timestamp_field: "timestamp"
+      topicRegexps: |
+        [ "lsst.scimma" ]
+      debug: true
+      offset: newest
```

Comment:
In general is better to use the default here `offset: oldest`, which means that when the connector restarts it will continue consuming the earliest message available as opposed to the latest avoiding gaps in the data.